hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMBlockProtocolServer.java

```diff
@@ -335,6 +336,14 @@ public boolean addSCM(AddSCMRequest request) throws IOException {
     }
   }
 
+  @Override
+  public String getNetworkTopology() {
+    String schemaFile = conf.get(
```

Comment:
We should serialize the file itself and send it or define a protobuf definition of the schema used in the file.

Reply:
> As per discussion, we would be skipping serving the schema file from SCM to OM as the schema file should be present under OM’s etc/hadoop/ directory even when OM and SCM are deployed on different nodes. Hence there should be no need for OM to access SCM’s etc/hadoop directory.

Resolving this discussion as per the above comment, thanks.